tests/pixie/utils_test/test_varint.py

```diff
@@ -138,8 +138,14 @@ def test_decode_encode_varint_number2(self):
     def _mocked_socket_bytes(self, int_value):
         mock_socket = mock.Mock()
         value = [
-            int.to_bytes(i, byteorder="little", length=1)
-            for i in encode_varint(int_value)
+            int.to_bytes(i, byteorder="little", length=1) for i in encode_varint(int_value)
         ]
         mock_socket.recv = lambda x: value.pop(x - 1)
         return mock_socket
+
+    def test_reads_from_socket_until_requested_length_fulfilled(self):
```

Comment:
Add a test with a packet larger than expected: 'res', 'ul', 'tsarina'. This should still return 'results'.